data/mods/Xedra_Evolved/monsters/monster.json

```diff
@@ -561,6 +561,7 @@
     "type": "MONSTER",
     "copy-from": "mon_gracke",
     "name": { "str": "gracken" },
-    "anger_triggers": [ "HURT" ]
+    "anger_triggers": [ "HURT" ],
+    "aggro_character": false 
```

Comment:
Pacifist gracken returns. 

One other thing I thought of: it’s probably worth it to add `”melee_training_cap": 5`, otherwise people will probably aggro them and abuse the melee skill of 10